### PR TITLE
feat(onboarding): Add completion_seen state

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Just adds a new field to the onboarding tasks model, for keeping track of what tasks have been 'seen' as completed.

### CODE REVIEW

File: src/sentry/api/endpoints/organization_onboarding_tasks.py [hunk 1]
```diff
@@ -26,11 +33,16 @@ def post(self, request, organization):
...
+
+        if status:
+            values["status"] = status
+            values["date_completed"] = timezone.now()
+        if completion_seen:
+            values["completion_seen"] = timezone.now()
+
         rows_affected, created = OrganizationOnboardingTask.objects.create_or_update(
-            organization=organization,
-            user=request.user,
```
Comment (wedamija) [inline src/sentry/api/endpoints/organization_onboarding_tasks.py]: Was it a bug that we had user here before?

**Review by wedamija — COMMENTED** [review]
This looks good, can you post the SQL that the migration generates?

File: src/sentry/api/endpoints/organization_onboarding_tasks.py (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/api/endpoints/organization_onboarding_tasks.py]: Yeah I think so.

**Comment by EvanPurkhiser** [comment]
```
BEGIN;
--
-- Add field completion_seen to organizationonboardingtask
--
ALTER TABLE "sentry_organizationonboardingtask" ADD COLUMN "completion_seen" timestamp with time zone NULL;
COMMIT;
```

I don't believe this will lock the table.

**Review by wedamija — APPROVED** [review]

**Pull request merged**